connector/rest/src/test/java/it/pagopa/selfcare/dashboard/connector/rest/client/PartyProcessRestClientTest.java

```diff
@@ -172,7 +168,7 @@ public void getInstitutionProducts_fullyNull() {
     @Test
     public void getInstitutionProducts_emptyResult() {
         // given and when
-        Products response = restClient.getInstitutionProducts(testCase2instIdMap.get(TestCase.EMPTY_RESULT));
+        Products response = restClient.getInstitutionProducts(testCase2instIdMap.get(TestCase.EMPTY_RESULT), null);
```

Comment:
set a list of values for the filter field _states_ as we did in the test _getInstitutionRelationships_emptyResult()_ (remember to change the wiremock stub as well)